projects/storefrontapp/src/app/app.module.ts

```diff
@@ -33,6 +33,9 @@ if (!environment.production) {
     BrowserTransferStateModule,
     StorefrontModule.withConfig({
       production: environment.production,
+      authentication: {
```

Comment:
Should we keep this in our repo if we mostly run e2e tests against servers without kyma addon enabled?